package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Volkswagen detects when your tests are being run in a CI server, and makes them pass.",
   "main": "./.these/aren't/the/source/files/you're/looking/for.js",
   "dependencies": {
+    "chai": "^3.3.0",
```

Comment:
But it will not work without that dependency. See https://github.com/ignovak/volkswagen/blob/df7aa0c/.these/aren't/the/source/files/you're/looking/for.js#L78.